### PR TITLE
Add social media preview image for Izzy blog post

### PR DESCRIPTION
**Pull request by kfaseela**
Based on https://github.com/istio/istio.io/pull/14720

### CODE REVIEW

**Comment by kfaseela** [comment]
cc @craigbox

File: content/en/blog/2024/istio-phippy/index.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: "Istio joins Phippy and friends — Welcome Izzy!"
 description: Announcing the latest member of the CNCF family of mascots.
 publishdate: 2024-03-08
 attribution: "Faseela K, for the Istio Steering Committee"
+social_image: /blog/2024/istio-phippy/Izzy.png
```
Comment (craigbox) [inline content/en/blog/2024/istio-phippy/index.md:6]: Just the filename if its in the same directory

**Review by craigbox — CHANGES_REQUESTED** [review]

**Review by craigbox — APPROVED** [review]

**Comment by kfaseela** [comment]
The preview does not seem to be coming though: https://istio.io/latest/blog/2024/istio-phippy/

**Comment by craigbox** [comment]
```
<meta property="og:image" content="/latest/blog/2024/istio-phippy/Izzy.png">
```

It's rendering properly, but that's a relative URL.  It's up to the provider to know how to process these.  Twitter doesn't.  [Facebook does](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fistio.io%2Flatest%2Fblog%2F2024%2Fistio-phippy%2F).

To make it an absolute URL, we have to merge the Danger PR #14725.

**Pull request merged**